ruby/reek/default.yml

```diff
@@ -140,7 +140,7 @@ detectors:
     enabled: false
     exclude: []
   UnusedPrivateMethod:
-    enabled: false
+    enabled: true
```

Comment:
`UnusedPrivateMethod ` does not work well for private method called with a specific syntax.
It is ok to be removed.